v3/keychron/q11/q11_us_knob_v1.0.json

```diff
@@ -0,0 +1,308 @@
+              "options": [0, 255],
+              "content": ["id_qmk_rgb_matrix_brightness", 3, 1]
+            },
+            {
+              "label": "Effect",
+              "type": "dropdown",
+              "content": ["id_qmk_rgb_matrix_effect", 3, 2],
+              "options": [
+                ["00. None", 0],
+                ["01. SOLID_COLOR", 1],
+                ["02. BREATHING", 2],
+                ["03. BAND_SPIRAL_VAL", 3],
+                ["04. CYCLE_ALL", 4],
+                ["05. CYCLE_LEFT_RIGHT", 5],
+                ["06. CYCLE_UP_DOWN", 6],
+                ["07. RAINBOW_MOVING_CHEVRON", 7],
+                ["08. CYCLE_OUT_IN", 8],
+                ["09. CYCLE_OUT_IN_DUAL", 9],
+                ["10. CYCLE_PINWHEEL", 10],
+                ["11. CYCKE_SPIRAL", 11],
+                ["12. DUAL_BEACON", 12],
+                ["13. RAINBOW_BEACON", 13],
+                ["14. JELLYBEAN_RAINDROPS", 14],
+                ["15. PIXEL_RAIN", 15],
+                ["16. TYPING_HEATMAP", 16],
+                ["17. DIGITAL_RAIN", 17],
+                ["18. REACTIVE_SIMPLE", 18],
+                ["19. REACTIVE_MULTIWIDE", 19],
+                ["20. REACTIVE_MULTINEXUS", 20],
+                ["21. SPLASH", 21],
+                ["22. SOLID_SPLASH", 22]
+              ]
+            },
+            {
+              "showIf": "{id_qmk_rgb_matrix_effect} > 1",
+              "label": "Effect Speed",
+              "type": "range",
+              "options": [0, 255],
+              "content": ["id_qmk_rgb_matrix_effect_speed", 3, 3]
+            },
+            {
+              "showIf": "{id_qmk_rgb_matrix_effect} != 0",
+              "label": "Color",
+              "type": "color",
+              "content": ["id_qmk_rgb_matrix_color", 3, 4]
+            }
+          ]
+        }
+      ]
+    }
+  ],
+  "customKeycodes": [
+    {"name": "Mission Control", "title": "Mission Control in macOS", "shortName": "MCtrl"},
+    {"name": "Launch Pad",      "title": "Launch Pad in macOS",      "shortName": "LPad"},
+    {"name": "Left Option",     "title": "Left Option in macOS",     "shortName": "LOpt"},
+    {"name": "Right Option",    "title": "Right Option in macOS",    "shortName": "ROpt"},
+    {"name": "Left Cmd",        "title": "Left Command in macOS",    "shortName": "LCmd"},
+    {"name": "Right Cmd",       "title": "Right Command in macOS",   "shortName": "RCmd"},
+    {"name": "Siri",            "title": "Siri in macOS",            "shortName": "Siri"},
+    {"name": "Task View",       "title": "Task View in windows",     "shortName": "Task"},
+    {"name": "File Explorer",   "title": "File Explorer in windows", "shortName": "File"},
+    {"name": "Screen Shot",     "title": "Screenshot in macOS",      "shortName": "SShot"},
+    {"name": "Cortana",         "title": "Cortana in windows",       "shortName": "Cortana"}
+  ],
+  "matrix": {"rows": 12, "cols": 9},
+  "layouts": {
+    "keymap": [
+      [
```

Comment:
```suggestion
      [
        {
          "c": "#aaaaaa"
        },
        "0,0\n\n\n\n\n\n\n\n\ne0",
        {
          "x": 0.25,
          "c": "#777777"
        },
        "0,1\nESC",
        {
          "c": "#cccccc"
        },
        "0,2",
        "0,3",
        "0,4",
        "0,5",
        {
          "c": "#aaaaaa"
        },
        "0,6",
        "0,7",
        {
          "x": 0.75
        },
        "6,0",
        "6,1",
        {
          "c": "#cccccc"
        },
        "6,2",
        "6,3",
        "6,4",
        "6,5",
        {
          "c": "#aaaaaa"
        },
        "6,6",
        "6,7",
        {
          "x": 0.25
        },
        "6,8\n\n\n\n\n\n\n\n\ne1"
      ],
      [
        {
          "y": 0.25
        },
        "1,0",
        {
          "x": 0.25
        },
        "1,1",
        {
          "c": "#cccccc"
        },
        "1,2",
        "1,3",
        "1,4",
        "1,5",
        "1,6",
        "1,7",
        {
          "x": 0.75
        },
        "7,0",
        "7,1",
        "7,2",
        "7,3",
        "7,4",
        "7,5",
        {
          "c": "#aaaaaa",
          "w": 2
        },
        "7,6",
        {
          "x": 0.25
        },
        "7,8"
      ],
      [
        "2,0",
        {
          "x": 0.25,
          "w": 1.5
        },
        "2,1",
        {
          "c": "#cccccc"
        },
        "2,2",
        "2,3",
        "2,4",
        "2,6",
        "2,7",
        {
          "x": 0.75
        },
        "8,0",
        "8,1",
        "8,2",
        "8,3",
        "8,4",
        "8,5",
        "8,6",
        {
          "c": "#aaaaaa",
          "w": 1.5
        },
        "8,7",
        {
          "x": 0.25
        },
        "8,8"
      ],
      [
        "3,0",
        {
          "x": 0.25,
          "w": 1.75
        },
        "3,1",
        {
          "c": "#cccccc"
        },
        "3,2",
        "3,3",
        "3,4",
        "3,5",
        "3,6",
        {
          "x": 0.75
        },
        "9,0",
        "9,1",
        "9,2",
        "9,3",
        "9,4",
        "9,5",
        {
          "c": "#777777",
          "w": 2.25
        },
        "9,7",
        {
          "x": 0.25,
          "c": "#aaaaaa"
        },
        "9,8"
      ],
      [
        "4,0",
        {
          "x": 0.25,
          "w": 2.25
        },
        "4,2",
        {
          "c": "#cccccc"
        },
        "4,3",
        "4,4",
        "4,5",
        "4,6",
        "4,7",
        {
          "x": 0.75
        },
        "10,0",
        "10,1",
        "10,2",
        "10,3",
        "10,4",
        {
          "c": "#aaaaaa",
          "w": 1.75
        },
        "10,5",
        {
          "x": 0.25,
          "c": "#777777"
        },
        "10,7"
      ],
      [
        {
          "c": "#aaaaaa"
        },
        "5,0",
        {
          "x": 0.25,
          "w": 1.25
        },
        "5,1",
        {
          "w": 1.25
        },
        "5,2",
        {
          "w": 1.25
        },
        "5,3",
        {
          "w": 1.25
        },
        "5,4",
        {
          "w": 2.25
        },
        "5,6",
        {
          "x": 0.75,
          "w": 2.75
        },
        "11,1",
        "11,2",
        "11,3",
        "11,4",
        {
          "x": 0.25,
          "c": "#777777"
        },
        "11,6",
        "11,7",
        "11,8"
      ]
```

Reply:
Hi, Here a gap is added between the left and right parts to make it look more like two parts.

Reply:
And the color of the key was adjusted to make it match the color of the standard keycap of the product.

Reply:
Gap added and colors matched, @lalalademaxiya1.